docs/UserGuide.md

```diff
@@ -5,17 +5,24 @@ title: User Guide
 * Table of Contents
 {:toc}
 --------------------------------------------------------------------------------------------------------------------
+## Motivation
 
+As a pet daycare owner, it takes a lot of effort to remember pets by their names,
+as well as remember their owner's names, phone numbers and addresses. In addition, some pets may have special dietary requirements and
+vet appointments that you have to take note off. With WoofAreYou, these things can be taken care
+of easily, so that you may focus on the things that **truly** matter: taking good care of the pets and ensuring their safety.
+--------------------------------------------------------------------------------------------------------------------
 # WoofAreYou
 
-WoofAreYou is a desktop application for pet daycare owners to handle the administrative information of their clients. 
-WoofAreYou is faster than any traditional platform such as Microsoft Excel or Google Sheets when you want to consolidate
+WoofAreYou is a desktop application for pet daycare owners like you to handle the administrative information of your clients. 
+WoofAreYou is faster than traditional platforms, such as Microsoft Excel or Google Sheets, when you want to consolidate
```

Comment:
I was thinking maybe can say `many traditional platforms`, to make our app sound cooler. Not sure though?